packages/expandable-card/src/ExpandableCard.tsx

```diff
@@ -269,6 +270,7 @@ const ExpandableCard = ({
               glyph="ChevronUp"
               size={24}
               fill={palette.gray.base}
+              tabIndex={0}
```

Comment:
⚠️ This should be an `IconButton` to have the right hover & focus styles

Reply:
This was going to be a separate ticket, but I guess I can just handle it here. I'll get to this tomorrow.